packages/ebsi-authorization-client/src/types/IEBSIAuthorizationClient.ts

```diff
@@ -0,0 +1,310 @@
+import { IAgentContext, ICredentialPlugin, IDIDManager, IKeyManager, IPluginMethodMap } from '@veramo/core'
+import { Format, PresentationDefinitionV2, PresentationSubmission } from '@sphereon/pex-models'
+
+/**
+ * The OpenID scope
+ * @readonly
+ * @enum {string}
+ */
+export enum OpenIDScope {
+  didr_write = 'openid%20didr_write',
```

Comment:
Hmzz, I don;t like that. The openId scope is litterally only "openid". Then EBSI is defining other scopes as well. Their eindpoints expect 2 scopes. Calling these OpenID scopes and then already combining them is not nice. Simply ensure the scope is a seperate enum and ensure the openid scope is always present when constructing the scope

Reply:
Changed the name of the enum to EBSIScope and added openid scope directly to the request